src/applications/disability-benefits/2346/containers/IntroductionPage.jsx

```diff
@@ -30,26 +30,26 @@ class IntroductionPage extends React.Component {
         <div className="process schemaform-process">
           <ol>
             <li className="process-step list-one">
-              <h3>Prepare</h3>
-              <p>To place an order, you'll need your:</p>
+              <h5>Prepare</h5>
+              <p>To place an order, you’ll need your:</p>
               <ul>
                 <li>Shipping address</li>
                 <li>Email address</li>
                 <li>Hearing aid information</li>
               </ul>
               <h4>What if I need help with my order?</h4>
               <p>
-                If you need help ordering hearing aid and betteries and
-                accessories, you cna contact the Devner Logistics Center (DLC)
-                at <a href="tel:303-273-6200">303-273-6200</a>
+                If you need help ordering hearing aid batteries and accessories,
+                you can call the DLC Customer Service Section at
```

Comment:
I think the "Denver Logistics Center" should be spelled out at least once before the abbreviation.